featurebyte/__init__.py

```diff
@@ -435,7 +435,6 @@ def list_deployments(
     "ValueBeyondEndpointImputation",
     "StringValueImputation",
     # feature & feature list version specific classes
-    "DefaultVersionMode",
```

Comment:
curious what's the reason behind removing this?

Reply:
Just found this unexpected change. Has reverted the changes here.